skops/saving_utils/model_card.py

```diff
@@ -0,0 +1,55 @@
+from modelcards import CardData, ModelCard
+from sklearn import set_config
+from sklearn.utils import estimator_html_repr
+
+
+def _plot_estimator(model):
+    set_config(display="diagram")
```

Comment:
We should avoid setting any config on the global `sklearn` level, since it would interact and conflict with what the user has set. You can use `config_context` instead: https://scikit-learn.org/stable/modules/generated/sklearn.config_context.html#sklearn.config_context

Reply:
yeah I thought about setting config to different values at beginning and end of function itself which was dirty, thanks for pointing out!